java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -192,7 +196,7 @@ protected Object[] handleArrayOrNullResponse(Response response) throws RedisExce
     /**
      * @param response A Protobuf response
      * @return A map of <code>String</code> to <code>V</code>
-     * @param <V> Value type could be even map too
+     * @param <V> Value type, could be even map too
```

Comment:
nit: revert? Or update to: 
```suggestion
     * @param <V> Value type
```